AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/AWSDataStorePlugin+DataStoreSubscribeBehavior.swift

```diff
@@ -27,4 +27,40 @@ extension AWSDataStorePlugin: DataStoreSubscribeBehavior {
     public func publisher(for modelName: ModelName) -> AnyPublisher<MutationEvent, DataStoreError> {
         return publisher.filter { $0.modelName == modelName }.eraseToAnyPublisher()
     }
+
+    @available(iOS 13.0, *)
+    public func observeQuery<M: Model>(for modelType: M.Type,
+                                       where predicate: QueryPredicate? = nil,
+                                       sort sortInput: QuerySortInput? = nil)
+    -> AnyPublisher<DataStoreQuerySnapshot<M>, DataStoreError> {
+        reinitStorageEngineIfNeeded()
```

Comment:
this call to `reinitStorageEngineIfNeeded()` can be removed as it is again called inside the method on line 38